src/lib/graphql.ts

```diff
@@ -131,7 +131,7 @@ function initArweave(gateway: string) {
     config = LOCAL_GATEWAY_CONFIG;
   }
 
-  return Arweave.init(config);
+  return ((Arweave as any)?.default ?? Arweave).init(config);
```

Comment:
@ropats16 The reason it was updated to this: `((Arweave as any)?.default ?? Arweave).init(config);` everywhere due to the issues created by it in nextjs.

Reply:
I meant instead of directly returning `((Arweave as any)?.default ?? Arweave).init(config)` here, you do something like this:

```
let arweave: Arweave;
  const ArweaveClass = (Arweave as any)?.default ?? Arweave;
  ```
  
  Like you've done in the `transaction` and `wallet` files

Reply:
@ropats16 I have added a new commit to reflect on your suggestion.